decidim-dev/config/rubocop/capybara.yml

```diff
@@ -15,9 +15,10 @@ Capybara/NegationMatcher:
 Capybara/RedundantWithinFind:
   Enabled: false
 
+Capybara/SpecificActions:
+  Enabled: false
+
```

Comment:
I think this isn't necessary (there's no difference without this, so we should remove it)
```suggestion
```